micromanager_gui/main_window.py

```diff
@@ -524,41 +523,41 @@ def stage_x_left(self):
             -float(self.stage_wdg.xy_step_size_SpinBox.value()), 0.0
         )
         if self.stage_wdg.snap_on_click_checkBox.isChecked():
-            self.snap()
+            self._snap()  # pragma: no cover
 
     def stage_x_right(self):
         self._mmc.setRelativeXYPosition(
             float(self.stage_wdg.xy_step_size_SpinBox.value()), 0.0
         )
         if self.stage_wdg.snap_on_click_checkBox.isChecked():
-            self.snap()
+            self._snap()  # pragma: no cover
 
     def stage_y_up(self):
         self._mmc.setRelativeXYPosition(
             0.0,
             float(self.stage_wdg.xy_step_size_SpinBox.value()),
         )
         if self.stage_wdg.snap_on_click_checkBox.isChecked():
-            self.snap()
+            self._snap()  # pragma: no cover
 
     def stage_y_down(self):
         self._mmc.setRelativeXYPosition(
             0.0,
             -float(self.stage_wdg.xy_step_size_SpinBox.value()),
         )
         if self.stage_wdg.snap_on_click_checkBox.isChecked():
-            self.snap()
+            self._snap()  # pragma: no cover
 
     def stage_z_up(self):
         self._mmc.setRelativePosition(
             float(self.stage_wdg.z_step_size_doubleSpinBox.value())
         )
         if self.stage_wdg.snap_on_click_checkBox.isChecked():
-            self.snap()
+            self._snap()  # pragma: no cover
 
     def stage_z_down(self):
         self._mmc.setRelativePosition(
             -float(self.stage_wdg.z_step_size_doubleSpinBox.value())
         )
         if self.stage_wdg.snap_on_click_checkBox.isChecked():
-            self.snap()
+            self._snap()  # pragma: no cover
```

Comment:
I don't think these `no cover`s are good to include. This is legitimately untested and is something that should be tested. Though that can actually be implemented in #142

Reply:
@ianhi  I agree with you! I did add `no covers` here because will be gone in 142 (I'm now working on this PR too).